Solutions/CiscoWSA/Analytic Rules/CiscoWSADataExfiltration.yaml

```diff
@@ -0,0 +1,32 @@
+id: 32c460ad-2d40-43e9-8ead-5cdd1d7a3163
+name: Cisco WSA - Unexpected uploads
+description: |
+  'Detects unexpected file uploads.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CiscoWSA
+    dataTypes:
+      - CiscoWSAEvent

```

Comment:
The name 'CiscoWSAEvent' does not refer to any known table, tabular variable or function. Because of that validations are failing. You will have to add its schema in json format under [CustomTable](https://github.com/Azure/Azure-Sentinel/tree/master/.script/tests/KqlvalidationsTests/CustomTables) folder. Only than the validations would pass. Please update.